apps/src/studio/utils/commandQueue.js

```diff
@@ -0,0 +1,39 @@
+import { singleton as studioApp } from '@cdo/apps/StudioApp';
+import api from '../api';
+
+/**
+ * Execute the code for all of the event handlers that match an event name
+ * @param {string} name Name of the handler we want to call
+ * @param {boolean} allowQueueExtension When true, we allow additional cmds to
+ *  be appended to the queue
+ * @param {Array} extraArgs Additional arguments passed into the virtual JS
+ *  machine for consumption by the student's event-handling code.
+ */
+export function callHandler(name, allowQueueExtension, extraArgs = []) {
+  Studio.eventHandlers.forEach(function (handler) {
+    if (studioApp().isUsingBlockly()) {
```

Comment:
In order to make this testable I'd love for studio.js to use a different implementation of `callHandler` based on whether we're using Blockly or Droplet.  (Instead of calling one method that decides which implementation to use.)

What's the best way to do this?